opengrok-indexer/src/test/java/org/opengrok/indexer/configuration/ProjectTest.java

```diff
@@ -143,6 +147,10 @@ public void testMergeProjects1() {
 
         assertEquals(env.getTabSize(), p1.getTabSize());
         assertEquals(env.isNavigateWindowEnabled(), p1.isNavigateWindowEnabled());
+        assertEquals(env.getBugPage(), p1.getBugPage());
```

Comment:
Just FYI, recently found out about https://stackoverflow.com/questions/40796756/assertall-vs-multiple-assertions-in-junit5 which might be useful here? No need to change though

Reply:
Nice tip !